source/Fleet.cpp

```diff
@@ -526,11 +526,11 @@ void Fleet::Place(const System &system, Ship &ship)
 	auto center = ChooseCenter(system);
 	Point pos = center.first + Angle::Random().Unit() * OffsetFrom(center);
 
-	double velocity = ship.IsDisabled() ? 0. : Random::Real() * ship.MaxVelocity();
+	double speed = ship.IsDisabled() ? 0. : Random::Real() * ship.MaxSpeed(false);
```

Comment:
good catch, velocity is a vector, so I like this name change
but if we change it here, it should also be changed above

Reply:
I can return the compliment: good catch!